cv_next/components/ui/filters/modeSwitch.tsx

```diff
@@ -0,0 +1,26 @@
+
+export const ModeSwitch = ({
+    values,
+    value,
+    onChange,
+  }: {
+    value: string,
+    values: [string, string],
+    onChange: (newValue: string) => void,
+  }) => {
+    const isRightSide = value === values[1]
```

Comment:
What is happening here?
Please talk to me about this component

Reply:
Just note that as I mentioned in a different comment, this isn't used until the next version as we decided to scrap the whole CV versioning idea(for this time)

Reply:
OK so two options: a. don't include in the PR (i.e. delete the files from the remote), b. add all of the comments I wrote in these components in a `//TODO` line at the top of the files.
If they'll be merged in their current state, we will simply forget about these comments.